card/lib/card/subcards.rb

```diff
@@ -206,14 +206,8 @@ def fetch_subcard key
     end
 
     def prepend_plus name
-      case name
-      when Symbol
-        "+#{Card[name].name}"
-      when /^\+/
-        name
-      else
-        "+#{name}"
-      end
+      name = name.to_name
+      name =~ /^\+/ ? name : "+#{name}"
```

Comment:
The method treated symbols as codenames before this change, now it just converts them to names.